src/styles/map-components.css

```
@@ -152,12 +152,12 @@
   }
 
   .route-showcase__title {
-    @apply font-display font-extrabold text-2xl tracking-widest uppercase;
+    @apply font-sans font-extrabold text-2xl tracking-widest uppercase;
     text-shadow: 0 0 20px rgba(254, 234, 0, 0.3);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)